# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

rotateBehaviorActionGoal action_goal
rotateBehaviorActionResult action_result
rotateBehaviorActionFeedback action_feedback
